steam-client/src/main/java/com/avenga/steamclient/steam/asyncclient/steamuser/callback/MarketingMessageCallback.java

```diff
@@ -0,0 +1,61 @@
+package com.avenga.steamclient.steam.asyncclient.steamuser.callback;
+
+import com.avenga.steamclient.enums.EMarketingMessageFlags;
+import com.avenga.steamclient.generated.MsgClientMarketingMessageUpdate2;
+import com.avenga.steamclient.model.GlobalID;
+import com.avenga.steamclient.model.steam.user.MarketingMessage;
+import com.avenga.steamclient.steam.asyncclient.callbackmanager.BaseCallbackMessage;
+import com.avenga.steamclient.util.stream.BinaryReader;
+import com.avenga.steamclient.util.stream.MemoryStream;
+import lombok.Getter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+/**
+ * This callback is fired when the client receives a marketing message update.
+ */
+@Getter
+public class MarketingMessageCallback extends BaseCallbackMessage {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MarketingMessageCallback.class);
+
+    private Date updateTime;
```

Comment:
khm. its 2k20. use of java.util.Date can cause a cancer

Reply:
Fixed in all places